<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotateVerticesWidget</class>
 <widget class="QWidget" name="RotateVerticesWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set position</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="floating" stdset="0">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <layout class="QFormLayout" name="formLayout1">
     <item row="0" column="0">
      <widget class="QLabel" name="angleL">
       <property name="autoFillBackground">
       </property>
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="angle">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item row="0" column="0">
    <widget class="QLabel" name="corlabel">
     <property name="autoFillBackground">
     </property>
     <property name="text">
      <string>Center of rotation</string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QFormLayout" name="formLayout2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCX">
       <property name="autoFillBackground">
       </property>
       <property name="text">
        <string>X coord</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="centerX">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCY">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Y coord</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="centerY">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCZ">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Z coord</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="centerZ">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item row="0" column="0">
    <widget class="QLabel" name="axlabel">
     <property name="autoFillBackground">
     </property>
     <property name="text">
      <string>Axis</string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonX">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonY">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonZ">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QFormLayout" name="formLayout3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAX">
       <property name="autoFillBackground">
       </property>
       <property name="text">
        <string>X coord</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="axisX">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>1.0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAY">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Y coord</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="axisY">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelAZ">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Z coord</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="axisZ">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly|Qt::ImhPreferNumbers</set>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="placeholderText">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>

    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSelectV">
       <property name="text">
        <string>Select Vertices</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>48</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout3">
     <item>
      <spacer name="horizontalSpacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionApply">
   <property name="text">
    <string>Apply</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>centerX</tabstop>
  <tabstop>centerY</tabstop>
  <tabstop>centerZ</tabstop>
  <tabstop>buttonX</tabstop>
  <tabstop>buttonY</tabstop>
  <tabstop>buttonZ</tabstop>
  <tabstop>axisX</tabstop>
  <tabstop>axisY</tabstop>
  <tabstop>axisZ</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
